Camera2Basic/app/src/main/java/com/example/android/camera2/basic/fragments/CameraFragment.kt

```diff
@@ -453,6 +453,11 @@ class CameraFragment : Fragment() {
         imageReaderThread.quitSafely()
     }
 
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _fragmentCameraBinding = null
```

Comment:
nit: reverse the sequence of these two lines?

Reply:
Done